src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/Models/ServiceObjectiveName.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.17.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Sql.Models
+{
+
+    /// <summary>
+    /// Defines values for ServiceObjectiveName.
+    /// </summary>
+    public static class ServiceObjectiveName
+    {
+        public const string Basic = "Basic";
+        public const string S0 = "S0";
+        public const string S1 = "S1";
+        public const string S2 = "S2";
+        public const string S3 = "S3";
+        public const string P1 = "P1";
+        public const string P2 = "P2";
+        public const string P3 = "P3";
```

Comment:
The Azure SDK team had me add this.  I know it's hard to keep up-to-date, but that's why I have it as model-as-string, so values not in this list can also be specified.